x-pack/plugins/security_solution/public/flyout/entity_details/host_details_left/index.tsx

```diff
@@ -19,25 +22,51 @@ export interface HostDetailsPanelProps extends Record<string, unknown> {
   isRiskScoreExist: boolean;
   name: string;
   scopeId: string;
+  hasMisconfigurationFindingsIndex?: boolean;
+  hasMisconfigurationFindingsForThisQuery?: boolean;
+  path?: {
+    tab?: EntityDetailsLeftPanelTab;
+  };
 }
 export interface HostDetailsExpandableFlyoutProps extends FlyoutPanelProps {
   key: 'host_details';
   params: HostDetailsPanelProps;
 }
 export const HostDetailsPanelKey: HostDetailsExpandableFlyoutProps['key'] = 'host_details';
 
-export const HostDetailsPanel = ({ name, isRiskScoreExist, scopeId }: HostDetailsPanelProps) => {
-  // Temporary implementation while Host details left panel don't have Asset tabs
-  const [tabs, selectedTabId, setSelectedTabId] = useMemo(() => {
+export const HostDetailsPanel = ({
+  name,
+  isRiskScoreExist,
+  scopeId,
+  path,
+  hasMisconfigurationFindingsIndex,
+  hasMisconfigurationFindingsForThisQuery,
+}: HostDetailsPanelProps) => {
+  const [selectedTabId, setSelectedTabId] = useState(
+    path?.tab === EntityDetailsLeftPanelTab.CSP_INSIGHTS
+      ? EntityDetailsLeftPanelTab.CSP_INSIGHTS
+      : EntityDetailsLeftPanelTab.RISK_INPUTS
+  );
+
+  const [tabs] = useMemo(() => {
     const isRiskScoreTabAvailable = isRiskScoreExist && name;
-    return [
-      isRiskScoreTabAvailable
-        ? [getRiskInputTab({ entityName: name, entityType: RiskScoreEntity.host, scopeId })]
-        : [],
-      EntityDetailsLeftPanelTab.RISK_INPUTS,
-      () => {},
-    ];
-  }, [name, isRiskScoreExist, scopeId]);
+    const riskScoreTab = isRiskScoreTabAvailable
+      ? [getRiskInputTab({ entityName: name, entityType: RiskScoreEntity.host, scopeId })]
+      : [];
+
+    // Determine if the Insights tab should be included
+    const insightsTab =
+      hasMisconfigurationFindingsIndex && hasMisconfigurationFindingsForThisQuery
```

Comment:
Is it really useful to have both `hasMisconfigurationFindingsIndex` and `hasMisconfigurationFindingsForThisQuery`? 

Most of the conditions seem to be checking for `hasMisconfigurationFindingsIndex && hasMisconfigurationFindingsForThisQuery`, in that case, wouldn't it be better to work only the `hasMisconfigurationFindingsForThisQuery`? since when `hasMisconfigurationFindingsForThisQuery` is true it means `hasMisconfigurationFindingsIndex` is also true.

Unless there is a use case for having only the `hasMisconfigurationFindingsIndex` that I'm missing